onfido/data.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import annotations
+
+import json
+
+
+def get_countries() -> list:
+    """Return a list of countries for consumer parsing."""
+    with open("onfido/data/supported_applicant_countries.json") as f:
+        countries = json.load(f)
+    return countries
+
+
+def check_supported_country(alpha3: str) -> bool:
+    """Check whether a given country is supported by OnFido."""
+    with open("onfido/data/supported_applicant_countries.json") as f:
+        countries = json.load(f)
+
+    if alpha3:
+        pick_country = next(
+            (country for country in countries if country["alpha3"] == alpha3), None
+        )
+        if pick_country:
+            return pick_country["supported_identity_report"]
+
+    return False
```

Comment:
This can be done in one pass:
```suggestion
return [
    c for c in countries
    if c["alpha3"] == alpha3
    and c["supported_identity_report"]
]
```